components/file_preview_modal/file_preview_modal_info/file_preview_modal_info.tsx

```diff
@@ -31,6 +31,10 @@ const FilePreviewModalInfo: React.FC<Props> = (props: Props) => {
     const channel = useSelector((state: GlobalState) => selectChannel(state, props.post.channel_id));
     const name = useSelector((state: GlobalState) => displayNameGetter(state, props.post.user_id, true));
 
+    if (!channel) {
+        return <div/>;
```

Comment:
Why empty div and not null?

Reply:
To not break flexboxes alignment

Reply:
Resolving this comment, since the fix that would crash the file preview (empty `channel/post`) is merged in upstream and these changes are being reverted